starter_code/express-spotify-app/app.js

```diff
@@ -0,0 +1,77 @@
+const express = require("express");
+const app = express();
+const hbs = require("hbs");
+const path = require("path");
+
+var SpotifyWebApi = require("spotify-web-api-node");
+app.set("view engine", "hbs");
+app.set("views", __dirname + "/views");
+app.use(express.static(path.join(__dirname, "public")));
+hbs.registerPartials(__dirname + "/views/partials");
+
+var clientId = "4c0a5b0bebd8473d82d5b08464e1412c",
```

Comment:
**This are sensitive data**, so I recommend always save as a enviorment variable. We already learn this, so in your future project always add this kind of data into your `.env` file, and make sure that your `.env` file is mentioned in the `.gitignore` file before to make any commit :grin: 

Then I will told you a funny not so funny story about this :wink: